### ISSUE TITLE
feat: ✨ improve `sd-select` focused option on popover open using tags

### ISSUE THREAD

**Issue by paulovareiro29** [solid-design-system/solid#1722]
## User Story
As a user of the Solid Design System using the `sd-select` component with the attributes `multiple` and `useTags`, when an option tag is focused using keyboard navigation and `space` is pressed to open, I would like the focused option on the popover to be the focused option tag.

### Suggested Solution
When an option tag is focused using keyboard navigation and `space` is pressed to open, the focused option on the popover should be the focused option tag.

## Subtasks
- [ ] Design_implementation_task
- [ ] Development_implementation_task
- [ ] Design_release_task

## DoR
- [ ] Item has business value
- [ ] Item all subtasks have been estimated by the team
- [ ] Item is clear and well-defined
- [ ] Item dependencies have been identified

## DoD
- [ ] All subtasks have been closed

**Comment by mariohamann** [comment]
@coraliefeil @MartaPintoTeixeira Does this make sense from A11y perspective? can you clarify with Carina or someone?
